### PR TITLE
Desktop implementation into master request

### PR DESCRIPTION
**Pull request by glitcher255**
Added desktop implementation view

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -902,3 +928,177 @@ body {
     top: 65%;
   }
 }
+
+@media only screen and (min-width: 768px) {
```
Comment (OyePriscilla) [inline style.css:933]: - [ ] You've really done a great work so far, the app is responsive but there is a little challenge you need to address. At `min-width: 768px` I expect to see all the elements in your app displayed properly. Please 🙏 ensure that all the sections are aligned properly. Thanks.

## For example, this is your header section @ `min-width: 768px`
Screenshot from 2022-11-25 09-04-14 (1)

File: style.css [hunk 2]
```diff
@@ -902,3 +928,177 @@ body {
...
+    gap: 25%;
+    justify-content: center;
+  }
+
+  #form {
+    top: 3850px;
+    flex-direction: row;
+    width: 100%;
+    align-items: baseline;
+  }
```
Comment (OyePriscilla) [inline style.css]: - [x] Please you may need to increase the gap between the two children elements according to the template. More so, there should be margin before the `get in touch` button. Please kindly effect this. 👍 

## Your Design
Screenshot from 2022-11-25 09-04-14 (1)

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @glitcher255 😃, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 

## Required Changes ♻️
- [x] Please kindly increase the gap between #item_nav and #ul elements so that we can have a design that is very close to the given template. 👍 
 

## Optional suggestions
- [optional] Please take note that your first two commits are not descriptive enough, and it suppose to be in sentence case. ℹ️

- [optional] I noticed you've been using ID as selector, It is not advisable to use IDs as CSS selectors because if another element in the page uses the same/similar style, you would have to write the same CSS again. Even if you don't have more than one element with that style right now, it might come later. Hence it is always advisable to use class.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Yazino12 — APPROVED** [review]
## Hi @glitcher255 🖐🏾,

You have done an outstanding job implementing the project requirements🎯, and there is nothing else left to say other than it's time to merge it 🚀🚀.

### To Highlight:
- Correct use of GitHub flow ✔
- Organized code ✔

# STATUS: APPROVED ✅✅✅
TURKISH
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions
- **_[OPTIONAL]_** In your future projects please make sure to create a descriptive pull request summary 🤝🏽.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**